core-model/src/main/java/com/sequenceiq/cloudbreak/domain/ClusterDefinition.java

```diff
@@ -33,12 +33,13 @@
     @Column(nullable = false)
     private String name;
 
-    @Column(nullable = false)
+    @Column(name = "blueprintText", nullable = false)
```

Comment:
can we rename this ?

Reply:
I would like to do the migration at the last step to keep backward compatible as late as possible